dotcom-rendering/src/web/components/Caption.tsx

```diff
@@ -208,14 +211,20 @@ const CameraIcon = ({ palette, format }: IconProps) => {
 	);
 };
 
-const VideoIcon = ({ palette, format }: IconProps) => {
+const VideoIcon = ({ palette, format, isMainMediaTest }: IconProps) => {
 	return (
 		<span
 			css={[
 				iconStyle(palette),
 				format.display === ArticleDisplay.Immersive &&
 					hideIconBelowLeftCol,
 				videoIconStyle,
+				// Temporary for AB test. Once complete this will be moved to palette
+				format.design === ArticleDesign.LiveBlog &&
+					isMainMediaTest &&
+					css`
+						fill: ${neutral[100]};
+					`,
```

Comment:
The risk here is that this code lives on. I think we've all universally had similar intentions to go back and fix things later but then not done it. It's really incredible how quickly and easily we move onto the next, more important, thing and leave these todos hanging.

Reply:
Yep, this is totally reasonable. Would having a removal PR ready before we merge these changes in abate these concerns?